CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Fix evaluation of parseMessage (#486)
```

Comment:
```suggestion
Error processing zero measures (#486)
```

I think this (or alternate improved wording) is better, as this way is closer to the user (parseMessage is an internal function).